presto-common/src/main/java/com/facebook/presto/common/predicate/TupleDomainFilterUtils.java

```diff
@@ -148,7 +152,7 @@ public static TupleDomainFilter toFilter(Domain domain)
 
     /**
      * Returns true is ranges represent != or NOT IN filter for double, float or string column.
```

Comment:
is --> if

Reply:
did it separately https://github.com/prestodb/presto/pull/22905